### ISSUE TITLE
Schema from var

### ISSUE THREAD

**Issue by hound672** [jsonschema-typed/jsonschema-typed#1]
Hello. I like your lib. But i'd like to load jsonschema from some var, not from file.
In my case I store some jsonschemas in one py file.
I thought to add logic that checs if there is .JSON ext in param. Example: JSONSchema['test_server/schema.json']
If there is not .JSON ext then should load schema from python module.

**Comment by erickpeirson** [comment]
Oh yeah, that's a good idea. So it would be like:

```python
data: JSONSchema['my_package.my_module.my_schema'] = dict(title='baz')
```

where ``my_package.my_module.my_schema`` is a schema dict?

**Comment by hound672** [comment]
Yes, smth like that. I tried to make it but faced with some troubles. 
So I write path to var like 'my_package/module/shema' it's not right but works for me.
Could I make pull request?

**Comment by erickpeirson** [comment]
Absolutely. Might make sense [somewhere in here](https://github.com/erickpeirson/jsonschema-typed/blob/85ee5b1e94fc5955108fab2af39d8b39ed9000cf/jsonschema_typed/plugin.py#L363); maybe using ``importlib.import_module`` to try to load the containing module?

**Comment by hound672** [comment]
Done. Please check it.
Also Python for expression like `data: JSONSchema['test_server/schema.json'] = dict(title='baz')` 
shows error:

> Traceback (most recent call last):
  File "test_server/sch_2.py", line 29, in <module>
    data: JSONSchema['test_server/schema.json'] = dict(title='baz')
TypeError: 'type' object is not subscriptable

Now I decided to wrap such expressions:
`from typing import TYPE_CHECKING
if TYPE_CHECKING:
`

**Comment by hound672** [comment]
Are you here?)

**Comment by felixvd** [comment]
I implemented that for a successor of this project, but the PR is lingering: https://github.com/inspera/jsonschema-typed/pull/8

@erickpeirson This package and [your Medium article](https://medium.com/@erick.peirson/type-checking-with-json-schema-in-python-3244f3917329) are still popular Google hits for this problem. Would you be willing to pass it on to a group of new maintainers, so it can live on?

**Comment by erickpeirson** [comment]
> I implemented that for a successor of this project, but the PR is lingering: https://github.com/inspera/jsonschema-typed/pull/8
> 
> @erickpeirson This package and [your Medium article](https://medium.com/@erick.peirson/type-checking-with-json-schema-in-python-3244f3917329) are still popular Google hits for this problem. Would you be willing to pass it on to a group of new maintainers, so it can live on?

@felixvd Certainly! What would you propose?

**Comment by felixvd** [comment]
@erickpeirson Thanks for your quick response, and sorry for my delayed one!

I would suggest to:
- Migrate this repo to an organization to manage roles. I invited you to the `jsonschema-typed` organization.
- Give maintainers access to the [jsonschema-typed package on pypi](https://pypi.org/project/jsonschema-typed/). Organizations can be requested on pypi.org, but I don't think they are meant for one-offs like this.

I would then add the updates from [this fork](https://github.com/inspera/jsonschema-typed) and my changes and take it from there.

**Comment by erickpeirson** [comment]
@felixvd Done

**Comment by felixvd** [comment]
@erickpeirson Thanks! Could you also grant Owner-level access so I can point the package to this repo?

**Comment by erickpeirson** [comment]
@felixvd done
